flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -2469,6 +2469,142 @@ void testCreateTableAsSelectWithPartitionKey() {
                                         "CREATE TABLE AS SELECT syntax does not support to create partitioned table yet."));
     }
 
+    @Test
+    void testReplaceTableAsSelectWithoutOptions() {
```

Comment:
I'd like to combine these test methods into two methods.
One is for ReplaceTableAs, and the other one if for CREATE OR REPLACE TABLE.

But I'll add comment in this combined method for which case to test.